src/fluentexpr.prw

```diff
@@ -22,21 +22,6 @@ Static Function Format( cString, aValues )
     Next
     Return cResult
 
```

Comment:
must add #include 'testsuite.ch' in this file

Reply:
since this PR is already merged, better create a new PR with suggested changes

Reply:
Fair enough. Should I create it?

Reply:
yes

Reply:
any help is welcome 👍